landa/water_body_management/report/catch_log_statistics/catch_log_statistics.py

```diff
@@ -128,18 +145,25 @@ def get_or_filters():
 		frappe.throw(_("You are not a member of any organization."))
 
 	if user_roles.intersection(REGIONAL_ROLES):
-		or_filters["regional_organization"] = member_data.regional_organization
-		or_filters["organization"] = ("like", f"{member_data.regional_organization}-%")
+		return query.where(
+			entry.regional_organization
+			== member_data.regional_organization
+			| entry.organization.like(f"{member_data.regional_organization}-%")
```

Comment:
```suggestion
			(entry.regional_organization == member_data.regional_organization)
			| (entry.organization.like(f"{member_data.regional_organization}-%"))
```
More readable what the OR is between